{{cookiecutter.project_slug}}/.envrc

```
@@ -0,0 +1,16 @@
+# read the secrets from 1Password 
+# strip whitespace from the value and base64 encode it
+AWS_ACCESS_KEY_SES_value=$(op read "op://{{cookiecutter.project_name}}/AWS_ACCESS_KEY_SES/token" | tr -d '[:space:]')
+export AWS_ACCESS_KEY_SES=$(echo "$AWS_ACCESS_KEY_SES_value" | base64 -w 0)
```

Please remove the base64 encoding and simply export the variable into the environment. Not all uses of these variables require them to be base64 encoded. I also added a comment on the manifest below to show how can use `stringData` in the manifest so we don't need to base64 encode these values.
```suggestion
export AWS_ACCESS_KEY_SES=$(op read "op://{{cookiecutter.project_name}}/AWS_ACCESS_KEY_SES/token")
```
Please update all the environment variables in this file.